src/v/config/configuration.cc

```diff
@@ -1545,7 +1546,35 @@ configuration::configuration()
       "Maximum capacity of rate limit accumulation"
       "in controller configuration operations limit",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      std::nullopt) {}
+      std::nullopt)
+  , kafka_throughput_limit_node_in_bps(
+      *this,
+      "kafka_throughput_limit_node_in_bps",
```

Comment:
Curious to why we designed two options, one for ingress other for egress instead of one unified total in/out throughput limit?

Reply:
There was a discussion about that in the RFC. One of the most essential reasons is that cloud providers tend to bill cluster ingress and egress separately, and they may even have different rates for them. There are also cases when ingress and egress of a cluster are asymmetric by design, and customer may want e.g. much more permissive egress limits, while restricing ingress to keep backpressure from spilling over into downstream.